src/screens/editor/container/editorContainer.tsx

```diff
@@ -561,10 +575,17 @@ class EditorContainer extends Component<any, any> {
             tags: fields.tags.join(' '),
           };
         }
-
+        
+        const meta = Object.assign({}, extractMetadata(draftField.body, thumbIndex), {
+          tags: draftField.tags,
+          beneficiaries,
```

Comment:
should also save schedule date and thumb index if not handled already.

Reply:
@aliseyalvi this feature is still missing....

https://user-images.githubusercontent.com/6298342/177179314-077ab6e8-637a-4b24-8d5f-d32c503e1ec4.mov